example/vm/config.yaml

```diff
@@ -242,13 +245,13 @@ service:
     pipelines:
         traces:
             receivers: [otlp]
-            processors: [batch, resourcedetection/env]
```

Comment:
is there any risk that changing the order of these could alter some expected output that depends on the order of operations? (assuming that these are executed sequentially and thats why we want this change?) i'm not familiar with these processors so I'm not sure what outputs we get here

Reply:
I changed the order to match the order of all the configs in the Helm charts, which always batch at the end (for example: https://github.com/lightstep/otel-collector-charts/blob/main/charts/otel-cloud-stack/values.yaml#L421)

Order does matter in general, telemetry passes through each processor sequentially. I'm pretty sure we want to batch right before sending to the exporter. @jaronoff97 can you please confirm?

Reply:
batch should always be at the end